styles/_tablet.scss

```diff
@@ -0,0 +1,68 @@
+
+
+.row {
+  flex-direction: column;
+}
+
+.grid-item {
+  width: 100%;
+}
+
+.shop-item {
+  flex-direction: row;
+}
+
+.shop-item__shirt {
+  width: 40%;
+}
+
+.shop-item__info {
+  width: 50%;
+}
+
+.shop-item__btns {
+  flex-direction: column;
+  align-items: flex-end;
+
+}
+
+.shop-btn {
+  display: flex;
```

Comment:
Using flex here will make your work lining up the menus MUCH harder. Don't flex where it's not needed